openreview/conference/templates/programchairWebfield.js

```diff
@@ -296,8 +296,12 @@ var getRegistrationForms = function() {
   var prefixes = [ REVIEWERS_ID, AREA_CHAIRS_ID ];
   var promises = _.map(prefixes, function(prefix) {
     return Webfield.getAll('/notes', {
-      invitation: prefix + '/-/.*Form'
-    });
+      invitation: prefix + '/-/.*',
+      signature: CONFERENCE_ID
+    })
+    .then(function(notes) {
+      return notes.filter(function(note) { return note.invitation.endsWith('Form'); })
+    })
```

Comment:
Are all the fields in the Form Notes necessary? We can use the `select` param to reduce the amount of data transferred to the client.

Reply:
good point, I will use `select=id`

Reply:
Done!